benchmarks/buiter_et_al_2016_jsg/exp_2_high_resolution.prm

```diff
@@ -0,0 +1,208 @@
+subsection Discretization
+  set Composition polynomial degree              = 2
+  set Temperature polynomial degree              = 2
+  set Stokes velocity polynomial degree          = 2
+  set Use locally conservative discretization       = false
+end
+
+# For restarting the model
+set Resume computation                           = auto
+subsection Checkpointing
+  set Steps between checkpoint             = 10
+end
+
+# Gravity model
+subsection Gravity model
+  set Model name               = vertical
+  subsection Vertical
+    set Magnitude               = 9.81
+  end
+end
+
+### Parameters describing the model
+
+# Model geometry (36 x 8 cm)
+ subsection Geometry model
+   set Model name               = box
+ 
+   subsection Box
+     set X repetitions               = 180
+     set Y repetitions               = 40
+     set X extent                   = 0.36
+     set Y extent                    = 0.08
+   end
+ end
+
+# Mesh refinement specifications 
+subsection Mesh refinement
+  set Initial adaptive refinement                = 0
+  set Initial global refinement                  = 2    #The resolution is 0.5 mm/cell
+  set Time steps between mesh refinement         = 0
+end
+
+# Boundary conditions & initial conditions
+
+# Composition boundary conditions
+subsection Boundary composition model
+  set Fixed composition boundary indicators         = bottom, left, right
+  set List of model names                 = initial composition
+end
+
+# Number and names of compositional fields
+subsection Compositional fields
+  set Number of fields             = 6
+  set Names of fields             = qsand,csand,bound,block,sticky_air,total_strain
+end 
+
+# Spatial domain of different compositional fields:
+## Quartz sand (qsand) has two layers with 1 cm thick per each.
+## Corundum sand (csand) is in the middle of the 2-layer quatz sand and is 1 cm thick.
+## The boundary 2-mm-thin layers (bound) sit on both the bottom of the domain and between the sand and 
+## rigid indenter block. The right boundary has a constant inflow velocity, which pushes a rigid block 
+## that approximates a mobile wall. Movement of the rigid block drives deformation in the sand layers.  
+## The boundary between the rigid block and boundary layers produces a sharp velocity discontinuity 
+## that localizes brittle deformation.
+## The sticky air layer is set on top of the sand layer and approximates a free surface.   
+
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Variable names                = x,y
+    set Function expression   = if((x<=0.35 & y<=0.012 & y>0.002) || (x<=0.35 & y>0.022 & y<=0.032), 1,0); \
+                              if((x<=0.35 & y>0.012 & y<=0.022), 1,0); \
+                              if((x>0.35 & x<=0.352 & y>0.002) ||(y<=0.002), 1, 0); \
+                              if(x>0.352 & y>0.002,1,0); \
+                              if(x<=0.35 & y>0.032,1,0); 0
```

Comment:
you could also align the `if`, the then-condition, and the else-condition here to make it easier to read this block